MIGRATIONS.md

```diff
@@ -3,7 +3,11 @@ Troubleshooting Migrations
 
 Migration to Version 1.17
 ---------------------------------
-Synchronizer APIs for listing sent and received transactions have been removed.  Clients should use `Synchronizer.clearedTransactions`, filtering on the field `TransactionOverview.isSentTransaction`.
+Synchronizer APIs for listing sent and received transactions have been removed.  Clients should use `Synchronizer.transactions`, filtering on the field `TransactionOverview.isSentTransaction`.
+
+Synchronizer APIs for pending transactions have been removed.  Clients should use `Synchronizer.transactions`, filtering on the field `TransactionOverview.isMined`
```

Comment:
I don't think this is correct since pending transactions can be mined but not yet confirmed.

Reply:
This has been updated